### DISCUSSION TITLE
Classic mod shows as "unranked" when selected

### DISCUSSION THREAD

**Discussion by marioman63** [ppy/osu#27339] [Q&A] [answered]
![image](https://github.com/ppy/osu/assets/76608775/ce2a047b-26ee-4f76-9a91-d89abf370200)
on version 2024.221.0, When selecting Classic mod, the UI shows that the current mod selection is unranked, but the score is still submitted, as can be seen on the results screen after a play with this mod:
![image](https://github.com/ppy/osu/assets/76608775/0c787e7d-7268-4cfc-8d68-74ae3cbc1c33)

**Comment by Stoppedpuma** [comment, marked as answer]
Unranked means PP cannot be gained, hover over the yellow boxed titled Unranked in mod select for more details.
